godot-core/src/registry/plugin.rs

```diff
@@ -5,38 +5,67 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+use crate::classes::Object;
```

Comment:
Maybe use the qualified version `classes::Object` in code, since `classes` is already imported below?